### PR TITLE
Styled Get Involved Boxes

### PR DESCRIPTION
**Pull request by Evavic44**
Had to create new utility classes on the HTML file to style the boxes. Still need to work on the custom responsiveness. @biggaji

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -300,30 +300,24 @@ <h1 class="partner_heading t-align-center mar-x-2">Partners</h1>
     </div>
     <div class="get_involved mar-x-5">
         <h1 class="head_text t-align-center mar-x-5">Get involved</h1>
-        <div class="ways-to-container d-flex flex-d-column f-align-center">
-            <div class="contribute t-align-center mar-x-5 padd-bottom-2">
+        <div class="ways-to-container d-flex flex-d-column f-align-center get_involved_wrapper">
+            <div class="contribute t-align-center mar-x-5 padd-bottom-2 box box-1">
                 <h3 class="padd-5 mar-bottom-4 ">Contribute code</h3>
```
Comment (biggaji) [inline index.html:305]: Try removing the class:  padd-5  from the h3 in the get involved section. @Evavic44

**Review by biggaji — CHANGES_REQUESTED** [review]

**Comment by biggaji** [comment]
> Had to create new utility classes on the HTML file to style the boxes. Still need to work on the custom responsiveness. @biggaji

Weldone, you're one step closer!

File: index.html (hunk 1 above)
Comment (Evavic44) [reply index.html]: Removed the ``pad-5`` class from each the ``h3`` headings. Please check it out.

**Review by biggaji — CHANGES_REQUESTED** [review]
Please  make the padding same as this in the image below.
image

**Comment by Evavic44** [comment]
Done.

**Pull request merged**